tests/integration/test_cloud_storage_api_client.py

```diff
@@ -0,0 +1,98 @@
+import os
+import pytest
+
+from datetime import datetime
+
+from mozmlops.cloud_storage_api_client import CloudStorageAPIClient
+
+def test_store_fetch_delete__nominal():
+    """
+    Test the integration to store a file, fetch a file, and delete a file. 
+    This test takes several seconds to run. Exclude integration tests except before release with the command:
```

Comment:
Happy to add some. Since this is more or less a pass-through API that calls the `google.cloud` storage functions,  what would we put in a unit test? Would we mock `google.cloud` and check that the `google.cloud` functions were called?

Reply:
Let's think about this separately in the related ticket (let's definitely not block the good work in this PR on this!)

Reply:
Added: https://mozilla-hub.atlassian.net/browse/DENG-3652